WindowsServerDocs/identity/ad-fs/troubleshooting/ad-fs-tshoot-certs.md

```diff
@@ -1,60 +1,58 @@
+|Event 374 - An error occurred while building the certificate chain for the claims provider trust encryption certificate.|The certificate has been revoked.</br></br>The certificate chain can't be verified.</br></br>The certificate is expired or isn't yet valid.|Ensure that the certificate is valid and hasn't been revoked.</br></br>Ensure that the CRL is accessible.|
+|Event 381 - An error occurred during an attempt to build the certificate chain for configuration certificate.|One of the certificates configured for use on the AD FS server is expired or has been revoked.|Ensure that all configured certificates haven't been revoked and haven't expired.|
+|Event 385 - AD FS detected that one or more certificates in the AD FS configuration database needs to be updated manually.|One of the certificates configured for use on the AD FS server has expired or is nearing its expiration date.|Update the expired or soon-to-expire certificate with a replacement. (If you're using self-signed certificates and automatic certificate rollover is enabled, this error can be ignored because it will self-resolve.)|
 |Event 387 - AD FS detected that one or more of the certificates that are specified in the Federation Service were not accessible to the service account that is used by the AD FS Windows Service.|The AD FS service account does not have read permissions to the private key of one or more configured certificates.|Ensure that the AD FS service account has read permission to the private key of all configured certificates.|
-|Event 389 - AD FS detected that one or more of your trusts require their certificates to be updated manually because they are expired, or will expire soon.|One of your configured partner's certificates has expired, or is about to expire. This can apply to either a claims provider trust or to a relying party trust.|If you have manually created this trust, update the certificate configuration manually. If you used federation metadata to create the trust, the certificate will update automatically as soon as the partner updates the certificate.|
-
-
-
+|Event 389 - AD FS detected that one or more of your trusts require their certificates to be updated manually because they are expired, or will expire soon.|One of your configured partner's certificates has expired, or is about to expire. This can apply to either a claims provider trust or to a relying party trust.|If you've manually created this trust, update the certificate configuration manually. If you used federation metadata to create the trust, the certificate will update automatically as soon as the partner updates the certificate.|
```

Comment:
_"This can apply to either a claims provider trust or to a relying party trust."_

Could you make this specific using a noun or by rewriting the sentence? Readers may have difficulty knowing what "this" refers to, especially when translated.